src/app/static/src/app/generated.d.ts

```diff
@@ -385,6 +385,64 @@ export interface DownloadStatusResponse {
       }
     | string)[];
 }
+export interface DownloadSubmitRequest {
+  notes?: {
+    project_notes?: Note;
```

Comment:
```suggestion
    project_note?: Note;
```
Should probably be singular as it is a single item, not an array. Here and anywhere else it shows up.

Reply:
This was generated from the schema @r-ash